pydarn/plotting/fan.py

```diff
@@ -52,12 +56,14 @@ def __str__(self):
     def plot_fan(cls, dmap_data: List[dict], ax=None,
                  scan_index: Union[int, dt.datetime] = 1,
                  ranges: List = [0, 75], boundary: bool = True,
-                 alpha: int = 0.5, parameter: str = 'v',
-                 lowlat: int = 30, cmap: str = None,
+                 line_color: str = 'black', alpha: int = 0.5,
+                 parameter: str = 'v', lowlat: int = 30, cmap: str = None,
                  groundscatter: bool = False,
                  zmin: int = None, zmax: int = None,
                  colorbar: bool = True,
-                 colorbar_label: str = '', title: bool = True):
+                 colorbar_label: str = '', radar_location: bool = False,
```

Comment:
``` suggestion
                 colorbar_label: str = '', radar_location: bool = True,
```

since boundary is true figured this should be true as well.